package.json

```diff
@@ -47,7 +47,7 @@
     "coveralls": "~2.11.8",
     "istanbul": "~0.4.2",
     "level-test": "~1.7.0",
-    "leveldown": "^1.4.4",
+    "leveldown": "~1.4.4",
```

Comment:
I'm a bit lost. What error? Removing 0.10 and 0.12 should make this work in theory.